database/migrations/2021_09_07_173057_create_announcements_table.php

```diff
@@ -17,7 +17,8 @@ public function up()
             $table->id();
             $table->text('title');
             $table->text('body');
-            $table->string('role')->default('Role');;
+            $table->string('role')->default('Role');
```

Comment:
Why is default set to 'Role'? It's better to make it nullable than to set it to a redundant value.